doc/getting-started.md

```diff
@@ -595,6 +595,8 @@ find ods-configuration -name '*.sample' -type f | while read NAME ; do mv "${NAM
 
 Now you will have to check the `.env` configuration files in `ods-configuration`. Change all values with the suffix `_base64` to a Base64 encoded value.  
 
+If you want later changes to the repositories, update `ods-configuration-sample` and copy the configuration-sample-script located in opendevstack.github.io to the parent folder of `ods-configuration-sample` and `ods-configuration`.  
```

Comment:
which script are you now referring to? have I overlooked a PR? Coud you provide a URL?

Reply:
The Script refers to the Script which will be in ods-core. I will update that to point to ods-core.

Reply:
It now seems to be ready for approval, isn't it

Reply:
Yes, should be ready for approval.